docs/contributing.rst

```diff
@@ -0,0 +1,40 @@
+Contributing
+============
+
+MSMBuilder is a collaborative project and we welcome all contributions.
+
+Users
+-----
+
+If you find a bug or have a question, consider opening an `issue
+<https://github.com/msmbuilder/msmbuilder/issues>`_. We're mainly grad
+student researchers, not software developers or support staff, so have
+patience if someone doesn't respond immediately. Don't be afraid to "bump"
+a thread if it's been a couple days.
+
+If you are an MSMBuilder user, we encourage you to "watch" the issue
+tracker. The developers will often solicit feedback about design decisions
+and features. Your input is important! You can also contribute by answering
+other users' questions.
+
+
+Developers
+----------
+
+To contribute a bug-fix or improvement, submit a pull request. Make sure
+the changed code is `pep8 <https://www.python.org/dev/peps/pep-0008/>`_
+compliant. Add a simple test case that would have failed but now passes
+with your contribution. Make a note of the change in the :ref:`changelog<changelog>`
+by editing ``docs/changelog.rst``. If you need help or clarification on any
+of these points, open a pull request and we'll be more than happy to help.
+
+To contribute a new feature, submit a pull request. The code should be
+`pep8 <https://www.python.org/dev/peps/pep-0008/>`_ compliant. Include a
+suite of tests to (1) verify your feature is working as intended and (2)
+will not be broken in the future. Describe the feature in the
+:ref:`changelog<changelog>` by editing ``docs/changelog.rst``. Provide literature
+citations if applicable. Add an example IPython notebook that demonstrates
+your new feature. It should run quickly and make a pretty plot.  If you
+need help or clarification on any of these points, open a pull request and
+we'll be more than happy to help.
+
```

Comment:
I think it would be useful to give some examples of bug-fixes/improvements vs. examples of features. For example I think a new featurizer has the potential to fall in either category. Where is the line drawn?